src/librustc/hir/mod.rs

```diff
@@ -916,8 +916,7 @@ impl Stmt_ {
     }
 }
 
-// FIXME (pending discussion of #1697, #2178...): local should really be
-// a refinement on pat.
+// FIXME: local should really be a refinement on pat.
```

Comment:
What this FIXME even mean? I'd rather remove it entirely (and its equivalent in `ast.rs` as well).
#1697 #2178 looks irrelevant.